mne_bids/path.py

```diff
@@ -72,17 +76,19 @@ class BIDSPath(object):
     """  # noqa
 
     def __init__(self, subject=None, session=None,
-                 task=None, acquisition=None, run=None, processing=None,
-                 recording=None, space=None, prefix=None, suffix=None):
+                 task=None, acquisition=None,
+                 run=None, processing=None,
+                 recording=None, space=None, root=None,
+                 kind=None, ext=None):
```

Comment:
looking at the signature as a whole, it kind of seems odd that `ext` is the only abbreviated param here now. Should be `extension`.